examples/end-to-end-session-based/02-End-to-end-session-based-with-Yoochoose-PyT.ipynb

```diff
@@ -112,10 +112,10 @@
      "name": "stderr",
```

Comment:
<p>Why do we have uncommented code?</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/NVIDIA-Merlin/Transformers4Rec/pull/680/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='37'/>